src/Js__JSON.res

```diff
@@ -1,13 +1,26 @@
 type t = Js.Json.t
 
+type jsonReviver
+external asJsonReviver: 'a => jsonReviver = "%identity"
+type jsonReplacer
+external asJsonReplacer: 'a => jsonReplacer = "%identity"
+
 @val external parseExn: string => t = "JSON.parse"
+@val external parseExnWithReviver: (string, jsonReviver) => t = "JSON.parse"
 @val external stringify: t => string = "JSON.stringify"
 @val external stringifyWithIndent: (t, @as(json`null`) _, int) => string = "JSON.stringify"
+@val external stringifyWithReplacer: (t, jsonReplacer) => string = "JSON.stringify"
+@val external stringifyWithReplacerAndIndent: (t, jsonReplacer, int) => string = "JSON.stringify"
 
 @val external parseToAnyExn: string => 'a = "JSON.parse"
+@val external parseToAnyExnWithReviver: (string, jsonReviver) => 'a = "JSON.parse"
 @val external stringifyAny: 'a => option<string> = "JSON.stringify"
 @val
 external stringifyAnyWithIndent: (t, @as(json`null`) _, int) => option<string> = "JSON.stringify"
+@val
+external stringifyAnyWithReplacer: (t, jsonReplacer) => option<string> = "JSON.stringify"
+@val
+external stringifyAnyWithReplacerAndIndent: (t, jsonReplacer, int) => option<string> = "JSON.stringify"
```

Comment:
Looks like the first argument should be `'a` instead of `t`

Reply:
Nice catch!

Reply:
Fixed on stringifyAnyWithIndent and the other methods added